packages/server/src/utils/KeyvDocumentNodeLRU.ts

```diff
@@ -0,0 +1,49 @@
+// Create ~about~ a 30MiB LRU. This is less than precise
+// since the technique to calculate the size of a DocumentNode is
+// only using JSON.stringify on the DocumentNode (and thus doesn't account
+// for unicode characters, etc.), but it should do a reasonable job at
+// providing a caching document store for most operations.
+//
+
+import LRUCache from 'lru-cache';
+import Keyv, { Store, type Options } from 'keyv';
+import type { DocumentNode } from 'graphql';
+
+// LRUCache wrapper to implement the Keyv `Store` interface.
```

Comment:
why is there no get? it just happens to work?

might be easier to read if this class has-a lru-cache instead of extends lru-cache. esp since we already know lru-cache API is about to change.

Reply:
Yeah, agree with the "has a cache" approach over extending the class.
cc3a87d33358fed639a487ff899f8c8c6f26da9d